mobile/lib/services/machine_learning/semantic_search/clip/clip_image_encoder.dart

```diff
@@ -1,15 +1,15 @@
-import "dart:typed_data" show Uint8List, Float32List;
+import "dart:typed_data" show Int32List, Uint8List;
 import "dart:ui" show Image;
 
 import "package:logging/logging.dart";
 import "package:onnx_dart/onnx_dart.dart";
 import "package:onnxruntime/onnxruntime.dart";
 import "package:photos/services/machine_learning/ml_model.dart";
-import "package:photos/utils/image_ml_util.dart";
 import "package:photos/utils/ml_util.dart";
 
 class ClipImageEncoder extends MlModel {
-  static const kRemoteBucketModelPath = "mobileclip_s2_image.onnx";
+  static const kRemoteBucketModelPath =
```

Comment:
is this model generating embeddings in a same space as before? if not so, existing embeddings will need invalidation, `++clipMlVersion`

Reply:
Yes it's essentially the same model operating in the same embedding space. It's just the preprocessing that has moved to a different layer. So totally compatible with existing embeddings.

Reply:
I see!